packages/web/src/common/store/tipping/sagas.ts

```diff
@@ -252,28 +238,23 @@ function* confirmTipIndexed({
     )
     try {
       const sdk = yield* getSDK()
+      const { data } = yield* call([sdk.full.tips, sdk.full.tips.getTips], {
+        txSignatures: [signature]
+      })
 
-      const senderUserId = Id.parse(sender.user_id)
-
-      const { data } = yield* call(
-        [sdk.full.users, sdk.full.users.getSupporter],
-        {
-          id: Id.parse(recipient.user_id),
-          supporterUserId: senderUserId,
-          userId: senderUserId
-        }
-      )
-
-      if (data) {
-        console.debug('Tip indexed')
+      if (data && data.length > 0) {
```

Comment:
nit: is `(data?.length > 0)` not sufficient here?

Reply:
Typescript no likey comparing undefined to a number